### ISSUE TITLE
Resume movie after playback

### ISSUE THREAD

**Issue by NikoPalad67140** [Lime3DS/Lime-3DS-Emulator#536]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

Basically, BizHawk allows you to resume TASing a game when you open a TAS file you haven't exported yet: an example would be in the WarioWare series: when you have gotten to the first Speed Up message, you save the TAS and take a break, and when you come back, the TAS can be resumed from where you left if off.

Since Citra (and by extension, Lime) doesn't support this, as closing the software saves the TAS for good without any means of resuming it, I would suggest several TASing features:

1. Being able to resume the TAS wherever you left it off after closing Lime, though the plyabacking feature should be kept for consistency.
2. Being able to create a new TAS from anywhere in the game, for example when you want to TAS only a specific portion of the game.

### Why would this feature be useful?

This feature can come in handy for larger games, rhythm games and/or for games which require a good amount of RNG-based decisions. If the software would be left as is, and the TAS can't be extended when closing the software, larger games can take way too much effort since you need to re-TAS them from scratch due to Lime saving the TAS for good when closing the software, preventing you from resuming once you get back to the game of your choice.

**Comment by TegronSpeedruns** [comment]
I would agree with that but Citra/Lime3DS and other forks are supposedly going to be banned on tasvideos.org once it's a core to BizHawk, besides that citra in the speedrunning communities including me agrees with one fact that Citra doesn't have good TAS tools, if you had to recode Lime3DS so that it had the same features as in BizHawk then sync would still be a problem for some games besides it is not known whether savestats would not be broken because even regardless of the non-deterministic aspect, if these loads are active, they break savestates, seriously enough to cause a crash during their creation and this problem can be even on this emulator.

**Comment by kage2051** [comment]
A recently pushed PR (https://github.com/azahar-emu/azahar/pull/656) introduced a 'workaround' setting to make async operations deterministic, though it does not fully address the determinism issues in some games. There is one more thing that should be stored in savestates and it's the NAND itself, including its state. Dolphin does this since 5.0-17850 (https://github.com/dolphin-emu/dolphin/pull/11184), not to mention it might be also useful for netplay.
